src/intervals/exact_literals.jl

```diff
@@ -53,6 +53,8 @@ struct ExactReal{T<:Real} <: Real
     ExactReal(value::T) where {T<:Real} = new{T}(value)
 end
 
+Base.iszero(x::ExactReal) = iszero(x.value)
```

Comment:
Maybe we should not hook too much into Base interface for `ExactReal` for now; so I would not define this function.
It may be safe to do so, but I think it is worth being very conservative as much as possible at the moment since `ExactReal` is relatively new.

Reply:
Removed.